lambdas/utils/filename_utils.py

```diff
@@ -0,0 +1,31 @@
+def extract_page_number(filename: str) -> int:
```

Comment:
Any tests been created for these utils? Sorry if I missed them

Reply:
Yes sorry my overlook and thanks for spotting it. I have just added unit tests for these utils :)